pkg/operator/starter.go

```diff
@@ -81,6 +90,25 @@ func RunOperator(ctx context.Context, controllerConfig *controllercmd.Controller
 	if err != nil {
 		return err
 	}
+	staticResourceController := staticresourcecontroller.NewStaticResourceController(
+		"OpenshiftAPIServerStaticResources",
+		v311_00_assets.Asset,
+		[]string{
+			"v3.11.0/openshift-apiserver/ns.yaml",
+			"v3.11.0/openshift-apiserver/apiserver-clusterrolebinding.yaml",
+			"v3.11.0/openshift-apiserver/svc.yaml",
+			"v3.11.0/openshift-apiserver/sa.yaml",
+			"v3.11.0/openshift-apiserver/trusted_ca_cm.yaml",
+		},
+		(&resourceapply.ClientHolder{}).WithKubernetes(kubeClient).WithAPIExtensionsClient(apiExtensionsClient),
+		operatorClient,
+		controllerConfig.EventRecorder,
+	).
+		AddEventHandler(kubeInformersForNamespaces.InformersFor(operatorclient.TargetNamespace).Core().V1().ConfigMaps().Informer().AddEventHandler).
+		AddEventHandler(kubeInformersForNamespaces.InformersFor(operatorclient.TargetNamespace).Core().V1().ServiceAccounts().Informer().AddEventHandler).
+		AddEventHandler(kubeInformersForNamespaces.InformersFor(operatorclient.TargetNamespace).Core().V1().Services().Informer().AddEventHandler).
+		AddEventHandler(kubeInformersForNamespaces.InformersFor(operatorclient.TargetNamespace).Rbac().V1().ClusterRoleBindings().Informer().AddEventHandler).
+		AddNamespaceEventHandler(kubeInformersForNamespaces.InformersFor(operatorclient.TargetNamespace).Core().V1().Namespaces().Informer().AddEventHandler, operatorclient.TargetNamespace)
```

Comment:
why we are reacting to a namespace?

Reply:
> why we are reacting to a namespace?

when the namespace is created or deleted, we want to recheck to see which resources need to have work done.